### PR TITLE
Update setup.py to contain current version

### PR DESCRIPTION
**Pull request by vildead**

### CODE REVIEW

**Comment by vildead** [comment]
@moustaphacheikh could you please take a look at this small change? Thanks!

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Validate version in setup.py
+      run: |
+        VERSION=$(python setup.py --version)
+        echo "Version in setup.py: $VERSION"
+        if [[ "$VERSION" != "$GITHUB_REF_NAME" ]]; then
```
Comment (moustaphacheikh) [inline .github/workflows/release.yml:24]: This means that the release format will change from **v1.8.1** to **1.8.1** ?

Comment (vildead) [reply .github/workflows/release.yml:24]: nice catch! I will add the `v` to the $VERSION variable

Comment (vildead) [reply .github/workflows/release.yml]: fixed @moustaphacheikh  :)

**Review by moustaphacheikh — APPROVED** [review]

**Pull request merged**